extensions/qute/runtime/src/main/java/io/quarkus/qute/i18n/MessageBundle.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.qute.i18n;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Denotes a message bundle interface.
+ * <p>
+ * Each method represents a single message:
+ * 
+ * <pre>
+ * &#64;MessageBundle
+ * interface MyBundle {
+ * 
+ *     &#64;Message("Hello {name}!")
+ *     String hello_world(String name);
+ * }
+ * </pre>
+ */
+@Retention(RUNTIME)
+@Target(TYPE)
+public @interface MessageBundle {
+
+    /**
+     * Constant value for {@link #locale()} indicating that the default locale of the Java Virtual Machine used to build the
+     * application should be used.
```

Comment:
I [hope that I] understand why this is, but it seems quite counter-intuitive. If it can't be the default locale of the _runtime_ JVM, then it should probably be highlighted in the user documentation more prominently.

Reply:
Good point...